src/artifact-utils.ts

```diff
@@ -46,7 +46,12 @@ export async function getArtifactRemoteURL(details: ElectronArtifactDetails): Pr
   const opts: MirrorOptions = details.mirrorOptions || {};
   let base = mirrorVar('mirror', opts, BASE_URL);
   if (details.version.includes('nightly')) {
-    base = mirrorVar('nightly_mirror', opts, NIGHTLY_BASE_URL);
+    if (opts.nightly_mirror) {
```

Comment:
This won't determine if they're using the old env var.  You should check if mirrorVar is truthy rather than just checking this option